pkg/controllers/disruption/consolidation_test.go

```diff
@@ -2302,6 +2302,38 @@ var _ = Describe("Consolidation", func() {
 			// and delete the old one
```

Comment:
We should think about writing our own ConsolidationValidator that pulls out the consolidateAfter so that we don't have to do the weird clock.Step() that we have everywhere throughout our testing

Reply:
Discussed offline, taking as a fast follow so as not to block release.